tools/deploy.sh

```diff
@@ -219,6 +220,11 @@ if [ "${DEPLOY_IRONIC}" == "true" ]; then
     else
       echo "INSPECTOR_REVERSE_PROXY_SETUP=${DEPLOY_TLS}" >> "${IRONIC_BMO_CONFIGMAP}"
     fi
+    if grep "RESTART_CONTAINER_CERTIFICATE_UPDATED" "${IRONIC_BMO_CONFIGMAP}" > /dev/null; then
```

Comment:
nit: use `grep -q` instead of `grep .. > /dev/null`

Reply:
Fixed! Thank you